### ISSUE TITLE
CogVideoX1.5-5B-I2V: The problem of consistency in video generation at any resolution

### ISSUE THREAD

**Issue by Alfonsobang** [wenqsun/DimensionX#24]
First of all, I would like to say that the work you are doing is fantastic, highly practical and far-reaching.
Here is one of my questions：
I used this beta version of comfyui node: https://github.com/kijai/ComfyUI-CogVideoXWrapper/tree/1.5_test
When I use the following two models, very different results appear for the same image ：
THUDM/CogVideoX-5b-I2V：When used with this lora at a fixed resolution (720*480): orbit_left_lora_weights.safetensors, the consistency of the video is maintained very well.
kijai/CogVideoX-5b-1.5-I2V：Used with this lora: orbit_left_lora_weights.safetensors, the consistency of the video is very poor.
This looks like a version update that has caused a deviation in the parameters used for this LORA, 
is there a solution? 
Thank you for your help.

**Comment by chenshuo20** [comment]
Hi, thank you for your interest! The issue mainly arises because we trained our Orbit LoRA on CogVideoX-5b-I2V instead of CogVideoX-5b-1.5-I2V. Directly transferring it may indeed cause some issues. We are considering training future versions of Orbit LoRAs using CogVideoX-5b-1.5-I2V as the base model.

**Comment by cheezecrisp** [comment]
> Hi, thank you for your interest! The issue mainly arises because we trained our Orbit LoRA on CogVideoX-5b-I2V instead of CogVideoX-5b-1.5-I2V. Directly transferring it may indeed cause some issues. We are considering training future versions of Orbit LoRAs using CogVideoX-5b-1.5-I2V as the base model.

This is really good to look forward to. BTW, any plan to train zoom in or zoom out lora?

**Comment by chenshuo20** [comment]
Certainly! This is already on our to-do list, and we'll be releasing them soon.

**Comment by yhd-ai** [comment]
> I would like to add that the orbit LoRAs work fantastically well with the CogVideoX-Fun 5b models as well. Which can be used as alternative for 1.5 if you want to use other resolutions/aspect ratios.
> 
> On 1.5 the LoRAs do work, just much weaker. Generally still well enough if you also prompt for the orbit.

Hi, I try to use CogVideoX-Fun 5b model by change pipeline to 
`pipe = CogVideoXImageToVideoPipeline.from_pretrained("alibaba-pai/CogVideoX-Fun-5b-InP", torch_dtype=torch.bfloat16)`

but I got error like this:
`raceback (most recent call last):
  File "/data0/haodong/DimensionX/src/gradio_demo/infer.py", line 53, in <module>
    video = pipe(image, prompt, use_dynamic_cfg=True)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/diffusers/pipelines/cogvideo/pipeline_cogvideox_image2video.py", line 822, in __call__
    noise_pred = self.transformer(
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1553, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1562, in _call_impl
    return forward_call(*args, **kwargs)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/diffusers/models/transformers/cogvideox_transformer_3d.py", line 476, in forward
    hidden_states = self.patch_embed(encoder_hidden_states, hidden_states)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1553, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1562, in _call_impl
    return forward_call(*args, **kwargs)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/diffusers/models/embeddings.py", line 715, in forward
    image_embeds = self.proj(image_embeds)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1553, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1562, in _call_impl
    return forward_call(*args, **kwargs)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/torch/nn/modules/conv.py", line 458, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "/data0/haodong/envs/dimensionx/lib/python3.10/site-packages/torch/nn/modules/conv.py", line 454, in _conv_forward
    return F.conv2d(input, weight, bias, self.stride,
RuntimeError: Given groups=1, weight of size [3072, 33, 2, 2], expected input[26, 32, 60, 90] to have 33 channels, but got 32 channels instead`
